### ISSUE TITLE
Terminal does not allow to edit previously used commands with screen reader normaly

### ISSUE THREAD

**Issue by justas2481** [microsoft/vscode#112589]
Issue Type: <b>Bug</b>

CTRL+` lets to open VSCode's integrated terminal and from there I can type and execute them, but when I type 1 command and press enter, after command is executed, if I would use up arrow to focus onto that previously written command, I am allowed only to choose it, but not edit it. By saying "not edit" I mean that I can actually edit that command, but NVDA screen reader doesn't read the characters that my cursor is focused on. On the other hand, if I just writing a new command, I can freely edit it without an issue.

VS Code version: Code 1.52.0 (940b5f4bb5fa47866a54529ed759d95d09ee80be, 2020-12-10T22:45:11.850Z)
OS version: Windows_NT x64 10.0.19042
Remote OS version: Linux x64 4.19.104-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz (8 x 2304)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.76GB (1.01GB free)|
|Process Argv|--crash-reporter-id aa2bbd19-3e03-4400-83dc-c73f0bdb6e71|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-20.04|
|OS|Linux x64 4.19.104-microsoft-standard|
|CPUs|Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz (8 x 2304)|
|Memory (System)|6.01GB (0.61GB free)|
|VM|0%|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-wsl|ms-|0.52.0
smarty-template-support|asw|1.8.0
vscode-intelephense-client|bme|1.5.4
vscode-html-css|ecm|1.1.5
prettier-vscode|esb|5.8.0
php-debug|fel|1.13.0
xml-format|mik|1.0.2
cpptools|ms-|1.1.3
php-docblocker|nei|2.1.0
php-debug|rob|1.14.0
go-to-method|tri|0.2.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221877
python763:30178808
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
python504:30227505
vswsl492:30211401
wsl2prompt:30224612
pythonvsdeb440:30233463
unusedprompt:30224610
folderexplorercf:30224615
openfilemenu:30224647
pythonvsded773cf:30236630

```

</details>

**Comment by meganrogge** [comment]
thanks for bringing this to our attention. 

when you say 
> but NVDA screen reader doesn't read the characters that my cursor is focused on

do you have any suggestion for how the terminal could handle this better or differently to enable the screen reader to read the characters?

**Comment by justas2481** [comment]
@meganrogge It should read chars the same way as it does when I just enter a new command and during the process editing it for example. What I mean is that if I am entering a new command (before pressing enter to issue it), I can use left / right arrows to move cursor by char or ctrl+left/right arrows to read command by word, but if I would press enter to execute it and then move with up arrow to focus on it (if I don't want to repeat it when I'm debugging for instance), I'm no longer able to read it by using arrows left/right and edit if if needed. I can edit, but it is just no longer being red by screen reader as it does with a new commands. So basically it should do the same as if I would entering a new command. WSL2 bash is a good example of how this should work.

**Comment by meganrogge** [comment]
Duplicate of #94708
